base/reducedim.jl

```diff
@@ -667,7 +667,7 @@ function findminmax!(f, Rval, Rind, A::AbstractArray{T,N}) where {T,N}
 end
 
 """
-    findmin!(rval, rind, A, [init=true]) -> (minval, index)
+    findmin!(rval, rind, A) -> (minval, index)
```

Comment:
The `init` argument exists. I guess you intended to use `;` to show it's a keyword argument? Same below.

Reply:
No I did this on purpose. https://github.com/JuliaLang/julia/issues/25188#issuecomment-353028482 where Stefan speculated init was a private keyword.

Reply:
Sorry, I had missed that. I'd remove the argument completely then, given that it doesn't seem to be used internally.

FWIW, `sortperm!` has an `initialized=false` argument which is somewhat similar (but reversed).

Reply:
Looks like `init` traces back to #6894. @timholy is there any reason to keep `init`?